testutil/testdatahandler.go

```diff
@@ -0,0 +1,105 @@
+package testutil
+
+import (
+	"path/filepath"
+)
+
+const (
+	TestDataFolderName     = "dcrdata-testdata"
+	TestDescriptorFileName = "test-description.json"
+)
+
+type TestDataHandler struct {
+	expected TestDescriptorJson
+}
+
+type TestDescriptorJson struct {
+	TestName                                string `json:"test_name"`
+	ExpectedBestBlockHeight                 int64  `json:"expected_block_height"`
+	ExpectedStakeInfoHeight                 int64  `json:"expected_stake_info_height"`
+	ExpectedBestBlockHash                   string `json:"expected_best_block_hash"`
+	ExpectedMD5OfRetrieveBlockFeeInfoString string `json:"expected_md5_of_retrieve_block_fee_info_string"`
+	FirstBlockNumber                        uint64 `json:"first_block_number"`
+	LastBlockNumber                         uint64 `json:"last_block_number"`
+}
+
+func (db *TestDataHandler) GetFirstBlockNumber() uint64 {
+	return db.expected.FirstBlockNumber
+}
+
+func (db *TestDataHandler) GetLastBlockNumber() uint64 {
+	return db.expected.LastBlockNumber
+}
+
+func (db *TestDataHandler) GetExpectedBestBlockHeight() int64 {
+	return db.expected.ExpectedBestBlockHeight
+}
+
+func (db *TestDataHandler) GetExpectedStakeInfoHeight() int64 {
+	return db.expected.ExpectedStakeInfoHeight
+}
+
+func (db *TestDataHandler) GetExpectedBestBlockHash() string {
+	return db.expected.ExpectedBestBlockHash
+}
+func (db *TestDataHandler) GetExpectedMD5OfRetrieveBlockFeeInfoString() string {
+	return db.expected.ExpectedMD5OfRetrieveBlockFeeInfoString
+}
+
+func LoadTestDataHandler(tag string) *TestDataHandler {
+	var tdb TestDataHandler
+	testDescriptorFile := PathToTestDescriptorFile(tag)
+
+	Log("reading:")
+
+	Log("testDescriptorFile", testDescriptorFile)
+	// uncomment this to produce example descriptor file
+	//ProduceExampleTestDescriptor(PathToTestDescriptorFile(tag))
+
+	testDescriptor := ReadTestDescriptorJson(testDescriptorFile)
+	tdb = TestDataHandler{
+		expected: testDescriptor,
+	}
+
+	return &tdb
```

Comment:
consider the simplification here too if you get rid of the unneeded struct type: `return ReadTestDescriptorJson(testDescriptorFile)`